src/utils/memoFn.js

```diff
@@ -0,0 +1,19 @@
+// let cachedResult = 0;
+
+export const memoFn = fn => {
+  const cache = {};
+
+  return (...args) => {
+    const key = args.join('-');
```

Comment:
This stringifies the args, but that means objects will get converted to `[object Object]`. Not sure the best way to handle it from a perf perspective, but this will be a bug if any memoized functions take objects as arguments.